soc/st/stm32/soc.yml

```diff
@@ -21,3 +21,6 @@ family:
     - name: stm32f103xe
     - name: stm32f105xc
     - name: stm32f107xc
+  - name: stm32mp1x
+    socs:
+    - name: stm32mp157cxx
```

Comment:
If A core is ported once day, this will have to be fixed, but it should be good for now